### PR TITLE
Feature interactive

### PR DESCRIPTION
**Pull request by fortuneonyeka**
In this branch I added the following functionalities:

 Add a new JavaScript file and import it as a module:
 it will contain methods related to the status updates (completed: true/false).
 Add event listener to the checkbox (change).
 Update items object's value for completed key upon user actions.
 Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 <p align="center">
     <img alt="Screenshot" src="./imgs/To-Do-List.png" width="500">
+    <img alt="Screenshot" src="./imgs/To-Do-List (checked).png" width="500">
 </p>
```
Comment (carlos-ssh) [inline README.md:21]: [OPTIONAL]
Keep your project clean and professional.
- Add your .DS_Store into your .gitignore.

File: src/index.js [hunk 2]
```diff
@@ -1,43 +1,53 @@
...
       </div>
       <i class="fas fa-ellipsis-v"></i>
     </li>`;
   });
 }
 
-window.addEventListener('load', renderList);
+function addToDo(e) {
+  e.preventDefault();
+  toDoList = JSON.parse(localStorage.toDoList);
```
Comment (carlos-ssh) [inline src/index.js:38]: I got this error message: (Kindly fix it)
Screen Shot 2021-08-19 at 18 28 35

- Your list of tasks isn't showing, i can't validate the interactivity of your project.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @fortuneonyeka, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (fortuneonyeka) [reply src/index.js:38]: I have checked through the project and ran it in different machines, I cant find the above errors in my project. please can I jump on a call with you to fix this?

**Review by mimipeshy — APPROVED** [review]
## Hi @Team 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] Since this project requires the use of ES6 syntax, you can consider using `arrow functions`  instead of the function keyword. ES6 introduces us to many great features like arrow functions. The arrow function is really awesome and makes your code more readable, more structured, and look modern

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**